### ISSUE TITLE
TypeError: Cannot destructure property 'organizations' of 'i' as it is null.

### ISSUE THREAD

**Issue by ackernaut** [ParabolInc/parabol#4967]
https://sentry.io/organizations/parabol/issues/2169591285/?referrer=github_plugin

```
TypeError: Cannot destructure property 'organizations' of 'i' as it is null.
  at c (./packages/client/modules/newTeam/NewTeam.tsx:71:10)
...
(7 additional frame(s) were not displayed)
```

This has affected 10 users in the last 2 months (as of Fri May 14)

### Acceptance criteria
- Triage and fix, or open a new issue with root cause to be fixed

### Estimated effort
- 4 hours

**Comment by tiffanyhan** [comment]
Sentry issue: [ACTION-PRODUCTION-8J4](https://sentry.io/organizations/parabol/issues/2515246499/?referrer=github_plugin)

**Comment by tiffanyhan** [comment]
Sentry issue: [ACTION-PRODUCTION-8J4](https://sentry.io/organizations/parabol/issues/2515246499/?referrer=github_plugin)

**Comment by mattkrick** [comment]
last seen 7 days ago, assuming fixed in 6.25.0
